test/gtest/uct/test_peer_failure.cc

```diff
@@ -627,20 +627,17 @@ class test_ucp_peer_failure_rma_zcopy : public test_uct_peer_failure
             scoped_log_handler slh(wrap_errors_logger);
             const size_t size = 128;
             mapped_buffer sendbuf(size, 0, *m_sender);
-            mapped_buffer recvbuf(size, 0, *m_receivers.front());
             ucs_status_t status;
 
             // Pretend peer failure by using invalid address
             if (is_put_op) {
                 status = uct_ep_put_zcopy(m_sender->ep(0),
                                           sendbuf.iov(), 1,
-                                          0xDEADBEAF, 0ul,
-                                          &m_dummy_comp);
+                                          UINT64_MAX, 0ul, &m_dummy_comp);
```

Comment:
minor: maybe define static const uint64_4 INVALID_ADDR = UINT64_MAX;
to document the reason we pass it

Reply:
fixed